File: core_backend/core_backend/Models/GoogleLog.cs
namespace core_backend.Models
{
    public class GoogleLog : BaseModel
    {
        public bool WasAllowed { get; set; }
        public string Query { get; set; }
        public float Lat { get; set; }
        public float Lon { get; set; }
        public long DateTime { get; set; }
        public string UserId { get; set; }
        public string Request { get; set; }
    }
}

File: backend/models/google_log.go
package models

import (
	"github.com/jinzhu/gorm"
)

type GoogleLog struct {
	gorm.Model

	WasAllowed bool
	Query      string
	Lat        float64
	Lon        float64
	DateTime   int64
	UserId     int64
	Request    string
}
